build.ps1

```diff
@@ -409,195 +435,234 @@ function Build-SwiftCorelibsXCTest($Arch)
   Move-Item -Force $PlatformInstallRoot\Developer\Library\XCTest-development\usr\lib\swift\windows\$LlvmArch\XCTest.swiftmodule $PlatformInstallRoot\Developer\Library\XCTest-development\usr\lib\swift\windows\XCTest.swiftmodule\$LlvmArch-unknown-windows-msvc.swiftmodule
 }
 
-function Build-SqliteX64
+function Build-SQLite($Arch)
 {
-  $Dest = "$SourceCache\sqlite-3.36.0\"
+  $ArchName = $Arch.ShortName
+  $Dest = "$SourceCache\sqlite-3.36.0"
 
   # Download the sources
   mkdir "S:\var\cache" -ErrorAction Ignore
-  if (-not(Test-Path -Path "S:\var\cache\sqlite-amalgamation-3360000.zip"))
+  if (-not (Test-Path -Path "S:\var\cache\sqlite-amalgamation-3360000.zip"))
   {
     curl.exe -sL https://sqlite.org/2021/sqlite-amalgamation-3360000.zip -o S:\var\cache\sqlite-amalgamation-3360000.zip
   }
 
-  if (-not(Test-Path -Path $Dest))
+  if (-not (Test-Path -Path $Dest))
   {
     mkdir $Dest -ErrorAction Ignore
     ."$env:ProgramFiles\Git\usr\bin\unzip.exe" -j -o S:\var\cache\sqlite-amalgamation-3360000.zip -d $Dest
     Copy-Item $SourceCache\swift-build\cmake\SQLite\CMakeLists.txt $Dest\
   }
 
   Build-CMakeProject `
-    -B $BinaryCache\sqlite-3.36.0.x64 `
+    -B $BinaryCache\sqlite-3.36.0.$ArchName `
     -D BUILD_SHARED_LIBS=NO `
     -D CMAKE_BUILD_TYPE=Release `
     -D CMAKE_INSTALL_PREFIX=$InstallRoot\sqlite-3.36.0\usr `
+    -D CMAKE_INSTALL_LIBDIR=lib/$ArchName `
     -D CMAKE_MT=mt `
     -G Ninja `
     -S $SourceCache\sqlite-3.36.0 `
     -Targets ($null, "install")
 }
 
-function Build-SwiftSystemX64
+function Build-SwiftSystem($Arch)
 {
+  $SwiftBuildDir = Get-ProjectBuildDir($Arch, 1)
```

Comment:
Careful, function call syntax does do what you think in powershell. It will pass a tuple as the first argument. I thought `Set-StrictMode` would catch this